packages/react/src/components/Datepicker/index.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+
+const DatepickerNavigation = React.forwardRef(
```

Comment:
Could this be moved into a prop or something?
For example, in the driver signup, we will want to render another component because we want to be able to select the month and the year

Reply:
@ugogo Done, let me know if it's ok :slightly_smiling_face: